nyx_client/nyx_client/data.py

```diff
@@ -28,6 +31,30 @@ class Data:
     in the Nyx system, including its metadata and content retrieval.
     """
 
+    name: str
```

Comment:
Could this be a non frozen dataclass?

Reply:
Yes, probably.

I think probably not frozen so you can change things on it. (I know right now there is no method for that but it would be nice to be able to update data by supplying an instance you've modified rather than the args.)

There are a couple of bits with logic (content type & url) - need to revisit those as well since would be easier if they didn't need that.

I'd rather leave it to next time (or until the other stuff we discussed on stand-up is complete).

Reply:
- `org` is implied by the client so shouldn't be needed. (the dataclass would instead have access to client)
- `url` - adding `buyer_org` is also always the same - shouldn't be needed there
- `content_type` - chopping off the end is unclear for anything other than IANA types (if we support those)